HACKER/HACKER/Sources/ViewControllers/ShoppingRankingViewController/Cells/RankingCollectionViewCell.swift

```diff
@@ -0,0 +1,271 @@
+//
+//  RankingCollectionViewCell.swift
+//  HACKER
+//
+//  Created by 김지수 on 2022/03/26.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+// MARK: - RankingCollectionViewCell
+class RankingCollectionViewCell: UICollectionViewCell {
+  
+  // MARK: - Components
+  let headerView = UIView()
+  let headerStackView = UIStackView()
+  let firstContainerView = UIView()
+  let firstUser = UIImageView()
+  let firstUserImage = UIImageView()
+  let firstUserNameLabel = UILabel()
+  let firstUserCommitLabel = UILabel()
+  let secondContainerView = UIView()
+  let secondUser = UIImageView()
+  let secondUserImage = UIImageView()
+  let secondUserNameLabel = UILabel()
+  let secondUserCommitLabel = UILabel()
+  let thirdContainerView = UIView()
+  let thirdUser = UIImageView()
+  let thirdUserImage = UIImageView()
+  let thirdUserNameLabel = UILabel()
+  let thirdUserCommitLabel = UILabel()
+  let separateView = UIView()
+  let rankingTableView = UITableView()
+  
+  let screenWidth = UIScreen.main.bounds.width-48
+  
+  // MARK: - LifeCycle
+  override func awakeFromNib() {
+    super.awakeFromNib()
+    register()
+    attribute()
+    layout()
+  }
+}
+// MARK: - Extensions
+extension RankingCollectionViewCell {
+  func register() {
+    self.rankingTableView.register(RankingTableViewCell.self, forCellReuseIdentifier: RankingTableViewCell.identifier)
+  }
+  func attribute() {
+    self.rankingTableView.delegate = self
+    self.rankingTableView.dataSource = self
+    self.rankingTableView.tableHeaderView = headerView
+  }
+  func layout() {
+    layoutHeaderView()
+    layoutHeaderStackView()
+    layoutFirstContainerView()
+    layoutFirstUser()
+    layoutFirstUserImage()
+    layoutFirstUserNameLabel()
+    layoutFirstUserCommitLabel()
+    layoutSecondContainerView()
+    layoutSecondUser()
+    layoutSecondUserImage()
+    layoutSecondUserNameLabel()
+    layoutSecondUserCommitLabel()
+    layoutThirdContainerView()
+    layoutThirdUser()
+    layoutThirdUserImage()
+    layoutThirdUserNameLabel()
+    layoutThirdUserCommitLabel()
+    layoutSeparateView()
+    layoutRankingTableView()
+  }
+  func layoutHeaderView() {
+    headerView.frame = CGRect(x: 0, y: 0, width: contentView.bounds.width, height: 325)
+  }
```

Comment:
```suggestion
  func layoutHeaderView() {
    headerView.frame = CGRect(x: 0, y: 0, width: contentView.bounds.width, height: 325)
    headerView.backgroundColor = .hackerWhite
  }
```
다크모드에서 애들이 까매져서 요 코드 추가~!
<img width="465" alt="image" src="https://user-images.githubusercontent.com/71601985/160806322-858b4610-1a36-4aa4-a88a-614b585786fc.png">